deploy/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
I think we should probably move the Service and ServiceMonitor out into the hack/test/deploy directory instead as this is handled automatically be the operator-sdk/olm